src/here-studio.ts

```diff
@@ -0,0 +1,298 @@
+                console.log("\nNo features are available to download");
+                //process.exit();
+            }
+
+            //Check if GeoSpaceData is blank -> if yes move on to next one else create the file with that name
+            let geoSpaceFileName = geoSpaceIDToCopy+".geojson";
+            await fs.writeFileSync(geoSpaceFileName, JSON.stringify(geoSpaceData));
+            console.log("Space '"+geoSpaceIDToCopy+"' downloaded locally")
+
+            //Copy the contents of the downloaded space to currentUser's XYZ spaces
+            let newSpaceData = await createSpace ({})//createNewSpaceAndUpdateMetadata(''+geoSpaceIDToCopy, ""+geoSpaceIDToCopy, {});
+            let currentGeoSpaceID = newSpaceData.id;
+
+            //Update the geospace id for current layer
+            currentLayer.geospace.id = currentGeoSpaceID;
+
+            let uploadOptions = {
+                title: geoSpaceFileName,
+                description: "GeoSpace created from HERE CLI",
+                file: geoSpaceFileName,
+                stream: true
+            }
+
+            //Upload it to current user's space
+            await uploadToXyzSpace (currentGeoSpaceID, uploadOptions);
+
+            //Clear cache and delete the file from temp repo
+            await fs.unlinkSync(geoSpaceFileName)
+
+            //Update the modified layer data.
+            updatedLayersData.push(currentLayer)
+        }
+    }
+
+    //Create a new project under current user with the settings of published projects
+    uri = "/project-api/projects";
+
+    //Update the layer data to cloned project - updatedLayersData
+    clonedProjectData.layers = updatedLayersData;
+
+    //Update the token for current project
+    clonedProjectData.rot = currentUsersToken;
+
+    let newProjectResponse = await execute(uri, "POST", cType, clonedProjectData, options.token);
+
+    let newProjectBody = newProjectResponse.body;
+    let studioBaseURL = "https://studio.here.com";
+    let clonedProjectURL = studioBaseURL+"/studio/project/"+newProjectBody.id;
+
+    //Viewer URL - /https://studio.here.com/viewer/?project_id=b1e3a9d4-116b-407b-b99f-17fbdf48d405
+    let viewerURL = studioBaseURL+"/viewer/?project_id="+newProjectBody.id
+    if (newProjectResponse.statusCode == 201) {
+        console.log("\n**** Successfully cloned ******")
+        console.log("Project cloned in your studio account : "+clonedProjectURL)
+        console.log("Viewer URL : "+viewerURL)
+    }
+}
+
+//Will capture URL Query string parameters from URL
+function getParameterByName (name: any, url: any) {
+    if (!url) return null;
+    name = name.replace(/[\[\]]/g, '\\$&');
+    var regex = new RegExp('[?&]' + name + '(=([^&#]*)|&|#|$)'),
+        results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, ' '));
+}
+
+async function showProject (id : any, options: any) {
+    const opn = require("opn");
```

Comment:
please use "open" library as that is the latest version

Reply:
Thanks for mentioning this, Have updated it to use open method.